{{cookiecutter.repo_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# How to contribute
+
+This is an OpenEdX repo, and we welcome your contributions!
+Here are a few guidelines to help your work flow smoothly to
```

Comment:
I'd lose the "Here are a few guidelines" sentence as there aren't actually any guidelines in this version of the template.